lib/translate.js

```diff
@@ -0,0 +1,137 @@
+'use strict';
+var _ = require('lodash')
+  , parseOneAddress = require('email-addresses').parseOneAddress
+  , version = require('../package.json').version;
+
+var translationRules = [
```

Comment:
I'd rather we applied each rule set completely independently of the other to avoid any unexpected results.